src/test/java/org/apache/commons/compress/compressors/lz4/FramedLZ4CompressorRoundtripTest.java

```diff
@@ -104,15 +104,17 @@ private void roundTripTest(final String testFile, final FramedLZ4CompressorOutpu
 
     @Test
     public void test64KMultipleBlocks() throws IOException {
-        final ByteArrayOutputStream buffer = new ByteArrayOutputStream();
-        final byte[] expected = new byte[98304];
+        ByteArrayOutputStream buffer = new ByteArrayOutputStream();
+        byte[] expected = new byte[98304];
         new Random(0).nextBytes(expected);
-        try (FramedLZ4CompressorOutputStream compressor = new FramedLZ4CompressorOutputStream(buffer,
-                new FramedLZ4CompressorOutputStream.Parameters(FramedLZ4CompressorOutputStream.BlockSize.K64, true, false, false))) {
+        try (FramedLZ4CompressorOutputStream compressor = 
+                new FramedLZ4CompressorOutputStream(buffer, 
+                        new FramedLZ4CompressorOutputStream.Parameters(FramedLZ4CompressorOutputStream.BlockSize.K64, true, false, false))) {
             compressor.write(expected);
         }
         try (FramedLZ4CompressorInputStream sis = new FramedLZ4CompressorInputStream(new ByteArrayInputStream(buffer.toByteArray()))) {
-            assertArrayEquals(expected, IOUtils.toByteArray(sis));
+            final byte[] actual = IOUtils.toByteArray(sis);
+            assertArrayEquals(expected, actual);
```

Comment:
Hello @cpreisler 
Same question as before:
Why is this method different? Don't change it unless it is required. Non-required changes make PRs harder and longer to review.

Reply:
let me try to get this out of here. Again, when I committed my changes the merge was showing conflicts with master, so that's how it got here.

Reply:
This is gone. From my perspective this is good to go. Please let me know if you want any additional changes. I'm available tonight and tomorrow.